elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -8,27 +18,27 @@
                     </a>
                 </li>
             </ul>
-            <ul id="{{answer_name}}-options" name="{{answer_name}}" class="pl-order-blocks-connected-sortable {{inline}}" indent={{max_indent}}>
+            <ul id="order-blocks-options-{{uuid}}" name="{{answer_name}}" class="pl-order-blocks-connected-sortable {{inline}}" indent={{max_indent}}>
```

Comment:
I believe the `indent` attribute is no longer in use and can be removed.
```suggestion
            <ul id="order-blocks-options-{{uuid}}" name="{{answer_name}}" class="pl-order-blocks-connected-sortable {{inline}}">
```

Reply:
nice catch